spec/view_helper/links_spec.rb

```diff
@@ -122,4 +122,25 @@
       end
     end
   end
+
+  context 'displaying image_src url' do
+    it 'should not display image_src url by default' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.